src/Dialog/index.js

```diff
@@ -96,7 +96,7 @@ const Dialog = ({
   const dialogJSX = (
     <CSSTransition timeout={1} classNames="nds-dialog-transition" appear in>
       <div className="nds-shim--dark" ref={shimRef} onClick={handleShimClick}>
-        <FocusLock autoFocus={false}>
+        <FocusLock autoFocus={false} className="nds-dialog-focuslock">
```

Comment:
Because this component inserts a div, we must use this element to set height constraints on the inner `nds-dialog` element